spec/models/update_user_spec.rb

```diff
@@ -45,10 +46,46 @@
     let(:user_id) { user.id }
     let(:user) { current_user }
 
+    context 'user updates DUNS to invalid DUNS number' do
+      it 'raises validation error' do
+        old_duns_number = user.duns_number
+        params = ActionController::Parameters.new(
+          id: user_id,
+          user: {
+            duns_number: "BAD"
+          }
+        )
+
+        updater = UpdateUser.new(params, current_user)
+        updater.save
+
+        expect(updater.errors).to eq(
+          'DUNS number format is invalid'
+        )
+        expect(user.duns_number).to eq old_duns_number
+      end
+    end
+
+    context 'user updates DUNS to nil' do
+      it 'does not raise validation error' do
+        params = ActionController::Parameters.new(
+          id: user_id,
+          user: {
+            duns_number: nil
+          }
+        )
+
+        updater = UpdateUser.new(params, current_user)
+        updater.save
+
+        expect(updater.errors).to eq('')
+      end
```

Comment:
we have a test for that in `spec/models/sam_account_reckoner_spec.rb` (L65) -- seems that is more the concern of the reckoner.